src/SamplesApp/SamplesApp.Droid/Properties/AssemblyInfo.cs

```diff
@@ -42,3 +42,4 @@
 [assembly: UsesPermission("android.permission.USE_FULL_SCREEN_INTENT")]
 [assembly: UsesPermission("android.permission.WRITE_EXTERNAL_STORAGE")]
 [assembly: UsesPermission("android.permission.CAMERA")]
+[assembly: UsesPermission("android.permission.RECORD_AUDIO")]
```

Comment:
You need to do the same for the net7.0-android build.

https://github.com/unoplatform/uno/blob/a3c1c76b951249c86242b8d0134f6d711de24dc1/src/SamplesApp/SamplesApp.netcoremobile/Android/AssemblyInfo.Android.cs